apps/web/src/components/Dropdown/index.tsx

```diff
@@ -6,57 +7,91 @@ import {
   Transition,
 } from "@headlessui/react";
 import { ChevronUpDownIcon, XMarkIcon } from "@heroicons/react/24/solid";
+import cn from "classnames";
 
+import useOverflow from "~/hooks/useOverflow";
 import { Option } from "./Option";
 
 export interface Option {
   value: string | number;
   label?: ReactNode;
   prefix?: ReactNode;
+  inputDisplay?: ReactNode;
```

Comment:
This is not an input so this name is a little bit confusing and not that descriptive. Let's rename it to something more consistent to the rest of the props. Maybe something like `selectedLabel`?

Reply:
Done 👍